internal/runbits/rtusage/rtusage.go

```diff
@@ -53,15 +53,6 @@ func NotifyRuntimeUsage(cfg *config.Instance, data dataHandler, orgName string)
 		return
 	}
 
-	if time.Now().Sub(cfg.GetTime(CfgKeyLastNotify)).Minutes() < float64(60) {
```

Comment:
Note I moved this down so that it only checks and records the timestamp if there is an actual notification to be sent.